pkg/instancegroups/instancegroups.go

```diff
@@ -88,16 +88,8 @@ func (c *RollingUpdateCluster) rollingUpdateInstanceGroup(ctx context.Context, c
 
 	if isBastion {
 		klog.V(3).Info("Not validating the cluster as instance is a bastion.")
-	} else if c.CloudOnly {
-		klog.V(3).Info("Not validating cluster as validation is turned off via the cloud-only flag.")
-	} else {
-		if err = c.validateCluster(); err != nil {
-			if c.FailOnValidate {
-				return err
-			}
-			klog.V(2).Infof("Ignoring cluster validation error: %v", err)
-			klog.Info("Cluster validation failed, but proceeding since fail-on-validate-error is set to false")
-		}
+	} else if err = c.maybeValidate("", 1); err != nil {
```

Comment:
What is reason for hard coding validateCount to 1 vs using c.validationCount which is 2 by default? It seems like a tiny speed gain for added code complexity passing around the value, also may not be what user expects if they have set a higher validationCount.

Reply:
The code hasn't recently made a change to the cluster, so it's not likely to flap from validated to failing. We don't want to hang around before we start waiting for it to inexplicably flap.

I doubt users who have set a higher validationCount would expect to have that count doubled between instance groups.

Reply:
Addon manifest will have been updated in S3, so CNIs for example may be updating on the existing nodes. On large clusters this can take a bit... That said I think this is something we can tweak/revisit if needed. I will LGTM for now if it is a problem we can address.